### PR TITLE
RATEST-112 - AddEditAppointmentBlockTest

### PR DESCRIPTION
**Pull request by Irenyak1**
## What I have done
Removed @ignore tag and the test can pass on both firefox and chrome in both with head and headless states.
 ## Ticket worked on
[RATEST-112](https://issues.openmrs.org/browse/RATEST-112)

### CODE REVIEW

**Comment by kdaud** [comment]
Investigate why the test is failing with `CI`

**Comment by Irenyak1** [comment]
Ok let me look into it

**Comment by kdaud** [comment]
> Ok let me look into it

You may need to `pull the trigger` to re-do the job. Wait until the server instance is up to do this. Otherwise the current does not tell the truth about the test

**Review by kdaud — COMMENTED** [review]
Do you still need the package `import org.junit.Ignore;`

**Review by kdaud — COMMENTED** [review]
Remove redundant spaces for code readability. `Line Number 76`

**Comment by Irenyak1** [comment]
@kdaud apart from the above review, are there any other changes you would want me to implement before I could make another commit?

**Comment by kdaud** [comment]
Catch the comment above and update accordingly

**Comment by Irenyak1** [comment]
@kdaud there could be others, so it would be better to address them at once

**Comment by Irenyak1** [comment]
@kdaud  personally I feel that line 76 should not be removed, actually for code readability.

**Comment by Irenyak1** [comment]
@kdaud I have been looking through some of the merged tickets and majority of them have that space apart from the ones you have personally worked on. So i have realized that your style is kinda different.

**Comment by kdaud** [comment]
The line code has no harm at ALL.  
Just as you mentioned, some merged issues have it simply because of the reason above. However, IMO  for quality assurance with in our code base, such spaces should be omitted. Spaces are good when separating functions and also within a method that invokes many calls
cc: @Irenyak1 @sherrif10

File: ui-tests/src/test/java/org/openmrs/reference/AddEditAppointmentBlockTest.java [hunk 1]
```diff
@@ -12,13 +12,12 @@
...
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.openmrs.reference.groups.BuildTests;
 import static org.junit.Assert.assertTrue;
 
-@Ignore
+
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/AddEditAppointmentBlockTest.java:20]: This is good @Irenyak1  Get rid of this space as @kdaud  suggested  then all is well

**Review by sherrif10 — APPROVED** [review]
LGTM

**Comment by kdaud** [comment]
Well done @Irenyak1 !! 
If you do not mind, trigger the job to re-do the plan once again

**Pull request merged**